VAMobile/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -107,12 +125,7 @@ export function HomeScreen({}: HomeScreenProps) {
   const { loginTimestamp } = useSelector<RootState, AnalyticsState>((state) => state.analytics)
   const disRating = !!ratingData?.combinedDisabilityRating
   const monthlyPay = !!letterBeneficiaryData?.benefitInformation.monthlyAwardAmount
-  const unreadMessageCount =
-    foldersData?.inboxUnreadCount ||
-    foldersData?.data.find((folder) => folder.attributes.name === FolderNameTypeConstants.inbox)?.attributes
-      .unreadCount ||
-    0
```

Comment:
This extra bit of logic isn't needed, because this is already calculated in the query